docs/topic_guides/blocking/blocking_rules.md

```diff
@@ -0,0 +1,65 @@
+---
+tags:
+  - Blocking
+  - Performance
+---
+
+# The Challenges of Record Linkage
+
+One of the main challenges to overcome in record linkage is the **scale** of the problem.
+
+The number of pairs of records to compare grows using the formula $\frac{n\left(n-1\right)}2$, i.e. with (approximately) the square of the number of records, as shown in the following chart:
+
+![](../../img/blocking/pairwise_comparisons.png)
+
+For example, a dataset of 1 million input records would generate around 500 billion pairwise record comparisons.
+
+So, when datasets get bigger the amount of computational resource gets extremely large (and costly). In reality, we try and reduce the amount of computation required using **blocking**.
+
+## Blocking
+
+Blocking is a technique for reducing the number of record pairs that are considered by a model.
+
+Considering a dataset of 1 million records, comparing each record against all of the other records in the dataset generates ~500 billion pairwise comparisons. However, we know the vast majority of these record comparisons won't be matches, so processing the full ~500 billion comparisons would be largely pointless (as well as costly and time-consuming).
+
+Instead, we can define a subset of potential comparisons using **Blocking Rules**. These are rules that define "blocks" of comparisons that should be considered. For example, the blocking rule:
+
+`"l.first_name = r.first_name and l.surname = r.surname"` 
+ 
+will generate pairwise record comparisons amongst pairwise comparisons where first name and surname match.
+
+Within a Splink model, you can specify multiple "blocks" through multiple Blocking Rules to ensure all potential matches are considered.
+
+???+ "Further Reading"
+
+    For more information on blocking, please refer to [this article](https://toolkit.data.gov.au/data-integration/data-integration-projects/probabilistic-linking.html#key-steps-in-probabilistic-linking)
+
+### Choosing Blocking Rules
+
+The blocking process is a compromise between the amount of **compuational resource** used when comparing records and **capturing all true matches**. 
```

Comment:
Might be worth saying something like:

"The aim of blocking rules is to have high recall.  Between them, your blocking rules should aim to recover all matching record pairs.

Precision is less important:  record comparisons that 'pass' the blocking rules are then put forward to the scoring/prediction step.  So it doesn't matter if the blocking rules select some (or even many) record pairs which are not matches.

Reply:
I have reworded this a bit and added